packages/ui-kit/src/components/IconButton/IconButton.tsx

```diff
@@ -7,15 +7,28 @@ import { Button, ButtonProps, ButtonLabel } from "../Button/Button";
 export interface IconButtonProps extends Omit<ButtonProps, "variant"> {
   icon: ReactNode;
   reverse?: boolean;
+  variant?: "bare";
 }
 
 export const IconButton: VFC<IconButtonProps> = ({
   label,
   icon,
   className,
+  variant,
   reverse,
   ...rest
 }) => {
+  if (variant === "bare") {
+    return (
+      <button
+        aria-label={typeof label === "string" ? label : undefined}
```

Comment:
So if label passed wasn't passed it's `undefined` right?

Reply:
Right.